junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/Node.java

```diff
@@ -120,6 +121,12 @@ default C execute(C context, DynamicTestExecutor dynamicTestExecutor) throws Exc
 	default void after(C context) throws Exception {
 	}
 
+	default void nodeSkipped(C context, TestDescriptor descriptor, SkipResult result) {
```

Comment:
The new `nodeSkipped()` and `nodeFinished()` methods both need `@since` and `@API` declarations as well as Javadoc.

Reply:
@junit-team/junit-lambda, we need to decide whether these new methods in `Node` should be _EXPERIMENTAL_, _STABLE_, or...

Reply:
**Team Decision:** Let's make them `MAINTAINED` like the other methods in `Node`.

Reply:
The new `nodeSkipped()` and `nodeFinished()` methods still need `@API` declarations (with `since = "1.4"`), and the `@since` tags need to be changed to `1.4`.